### PR TITLE
Update to remove tslint from bft - Partially Closes #4696

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR partially resolves #4696 

### How was it solved?

- Remove tslint from lisk-bft

### How was it tested?
- All the linting should work
- no source code changes

### CODE REVIEW

File: elements/lisk-bft/package.json [hunk 1]
```diff
@@ -24,8 +24,8 @@
 	"scripts": {
 		"clean": "./scripts/clean.sh",
 		"format": "prettier --write '**/*'",
-		"lint": "tslint --format verbose --project .",
-		"lint:fix": "npm run lint -- --fix",
+		"lint": "eslint . --ext .js,.ts",
+		"lint:fix": "eslint --fix --ext .js,.ts .",
```
Comment (nazarhussain) [inline elements/lisk-bft/package.json:28]: It's missing a `.` in command, but on other packages, we used different commands. 

```suggestion
		"lint:fix": "npm run lint -- --fix",
```

Comment (shuse2) [reply elements/lisk-bft/package.json:28]: it suppose to have the `dot` and it looks correct?

Comment (nazarhussain) [reply elements/lisk-bft/package.json:28]: Missed that dot at the end. Can we do it the way for lint command.

Comment (nazarhussain) [inline elements/lisk-bft/package.json:28]:
```suggestion
		"lint:fix": "eslint . --fix --ext .js,.ts",
```

Comment (shuse2) [reply elements/lisk-bft/package.json:28]: ill change all together in the last PR

**Pull request merged**